ovos_plugin_manager/utils/__init__.py

```diff
@@ -112,11 +112,17 @@ def find_plugins(plug_type: PluginTypes = None) -> dict:
                 if entry_point.name not in entrypoints:
                     LOG.debug(f"Loaded plugin entry point {entry_point.name}")
             except Exception as e:
-                LOG.error(f"Failed to load plugin entry point {entry_point}: "
-                          f"{e}")
+                if entry_point not in find_plugins._errored:
+                    find_plugins._errored.append(entry_point)
+                    # NOTE: this runs in a loop inside skills manager, this would endlessly spam logs
+                    LOG.error(f"Failed to load plugin entry point {entry_point}: "
```

Comment:
Should this be `LOG.exception`?

Reply:
we already include the exception in the log itself, no strong opinion, is the stack trace useful here?